example/setup-db.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
Maybe that bash script is unnecessary, especially since it requires to have sqlite installed.

A nodejs script we may call throught `npm run` would probably be neater. 
Or maybe even better: we may let the server create the database automatically if it does not exist yet, which is a quite simple solution.

Reply:
Done.